ui/pages/confirmations/hooks/alerts/transactions/useSigningOrSubmittingAlerts.test.ts

```diff
@@ -65,22 +65,26 @@ describe('useSigningOrSubmittingAlerts', () => {
   it('returns no alerts if no confirmation', () => {
     expect(
       runHook({
-        currentConfirmation: undefined,
         transactions: [TRANSACTION_META_MOCK],
       }),
     ).toEqual([]);
   });
 
   it('returns no alerts if no transactions', () => {
```

Comment:
Why do we need to mock for these scenarios?

Could we instead just update the `transactions` and `pendingApprovals` state passed to `runHook` since we're trying to avoid mocking to keep these tests less brittle?

Reply:
The test cases are checking for scenario like transaction on different chain, transaction in approved status. These scenario are not possible to create using state and normal code execution.

Reply:
These tests are checking scenarion, beyond login in hook `useSigningOrSubmittingAlerts`. These can be improved.